content/docs/extensions/model/index.md

```diff
@@ -0,0 +1,10 @@
+# Model extensions
+
+Model extensions add support for new types models that MLEM can covert into MLEM
+model objects in [`save` API method](/doc/api-reference/save)
```

Comment:
We should try to explain this at a higher level. What functionality do model extensions give to users? OK they extend MLEM model objects I think, how is that useful? Should we start with a recap of what a model object is (used for) first?

Reply:
I tried to address this in #188 in "Object Reference" section, e.g. see this page https://mlem-ai-new-docs-struct-hy95mv.herokuapp.com/doc/object-reference/model. If you have some feedback, you can post it here or do it in #188.

Reply:
Unable to leave feedback at this level in #188 atm, it's too huge.